src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,133 @@
+package wolox.training.models;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+
+@Entity
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(nullable = false, unique = true)
+    private String genre;
+
+    @Column(nullable = false, unique = true)
+    private String author;
+
+    @Column(nullable = false, unique = true)
+    private String image;
+
+    @Column(nullable = false, unique = true)
+    private String title;
+
+    @Column(nullable = false, unique = true)
+    private String subtitle;
+
+    @Column(nullable = false, unique = true)
+    private String publisher;
+
+    @Column(nullable = false, unique = true)
+    private String year;
+
+    @Column(nullable = false, unique = true)
+    private Integer page;
+
+    @Column(nullable = false, unique = true)
+    private String isbn;
+
+    private long getId() {
+        return id;
+    }
+
+    private String getGenre() {
+        return genre;
+    }
+
+    private String getAuthor() {
+        return author;
+    }
+
+    private String getImage() {
+        return image;
+    }
+
+    private String getTitle() {
+        return title;
+    }
+
+    private String getSubtitle() {
+        return subtitle;
+    }
+
+    private String getPublisher() {
+        return publisher;
+    }
+
+    private String getYear() {
+        return year;
+    }
+
+    private Integer getPage() {
+        return page;
+    }
+
+    private String getIsbn() {
+        return isbn;
+    }
+
+    private void setId(long id) {
```

Comment:
The IDs generated by the entity, thanks to the @GeneratedValue annotation, make it unnecessary to include the method "public void setId (Long id)".

Reply:
okay